src/hooks/useNodeViews.tsx

```diff
@@ -47,17 +47,19 @@ export function useNodeViews(
     []
   );
 
-  const reactNodeViews = useMemo(() => {
-    const nodeViewEntries = Object.entries(nodeViews);
-    const reactNodeViewEntries = nodeViewEntries.map(([name, constructor]) => [
-      name,
-      createReactNodeViewConstructor(constructor, registerPortal),
-    ]);
-    return Object.fromEntries(reactNodeViewEntries);
+  const wrappedNodeViews = useMemo(() => {
+    const nodeViewEntries = Object.entries(nodeViews ?? {});
+    const wrappedNodeViewEntries = nodeViewEntries.map(
+      ([name, constructor]) => [
+        name,
+        createReactNodeViewConstructor(constructor, registerPortal),
+      ]
+    );
+    return Object.fromEntries(wrappedNodeViewEntries);
   }, [nodeViews, registerPortal]);
 
   return {
-    nodeViews: reactNodeViews,
+    nodeViews: wrappedNodeViews,
     renderNodeViews: () => <NodeViews portals={portals} />,
```

Comment:
This probably doesn't need to be a function anymore if it's internal! I honestly forget why it was, but I don't think it's providing any functionality